bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/firmware/FirmwareUpdateService.java

```diff
@@ -84,18 +98,29 @@ public void run() {
         }
     };
 
-    protected void activate() {
```

Comment:
there is a default configuration to be used during activation... but I will also change the impl so that the config is given into the activate operation and the default is only used if no parmater is provided